tests/fullydistributed_grids/copy_distributed_tria_03.with_p4est=true.mpirun=5.output

```diff
@@ -0,0 +1,9 @@
+
+
+
+
```

Comment:
Your test doesn't actually produce any output. Where do these empty lines come from?

Reply:
I think each process prints one empty line per default.

Reply:
I see.

Reply:
If you don't need the output of every process, and it appears you don't, replace `MPILogInitAll all;` by `mpi_initlog();`